### ISSUE TITLE
xgcm authorship

### ISSUE THREAD

**Issue by vrx-** [xgcm/xgcm#231]
This comment is part of the JOSS review at openjournals/joss-reviews#2631

As part of the review process, the [guidelines](https://joss.readthedocs.io/en/latest/review_criteria.html#authorship) prompt me to confirm that the full list of paper authors seems appropriate and complete, and to raise an issue if it’s not obviously complete/correct. For example, 

* a couple of authors seem to have commits of a single typo fix (geosciz ) and yaml file for release (andersy005) which don't seem to be of obvious relevance for the development of the package
* there are other contributors, with both commits and/or also substantial input in raised issues, who are not part of the author list (raphaeldussin, dcherian, apatlpo)

I understand that contributions may also occur behind the scenes as substantial intellectual input, but as they are not obvious for the external reviewer I'm raising the issue to follow the JOSS guidelines so everything can be out in the open.

**Comment by rabernat** [comment]
Thanks @vrx-. Authorship was discussed internally in https://github.com/xgcm/xgcm/pull/181#issue-375778797.

The JOSS paper draft predates the contributions of @dcherian and @apatlpo, who have started contributing only in the past few months. Would it be okay to add them as authors at this stage? I think that would be great.

My initial approach was to offer authorship to everyone who has contributed, even if the contribution was small in terms of commits / lines of code. That appears to be acceptable based on the JOSS Guidelines:

> Ultimately, the authors themselves are responsible for deciding which contributions are sufficient for co-authorship, although JOSS policy is that purely financial contributions are not considered sufficient.

**Comment by vrx-** [comment]
There isn't anything in the guidelines that prevent adding authors at this stage, so as a reviewer I would encourage to add @dcherian and @apatlpo if they are willing.

**Comment by kthyng** [comment]
@rabernat and @jbusecke Yes you can add co-authors and should if appropriate, and yes you can choose who can be a co-author. Authorship for JOSS should be determined in a similar way to a traditional publication where all authors have contributed in some notable manner [and](https://joss.readthedocs.io/en/latest/submitting.html#authorship)

> co-authors agree to be accountable for all aspects of the work

**Comment by jbusecke** [comment]
I will open a PR to add both @dcherian and @apatlpo if they agree. 

If you could both send me a confirmation and your affiliation details (if available with ORCID like [here](https://raw.githubusercontent.com/xgcm/xgcm/master/paper.md)), that would be awesome.

**Comment by dcherian** [comment]
I am happy and grateful to be a coauthor on this paper

```
  - name: Deepak Cherian
    orcid: 0000-0002-6861-8734
    affiliation: 3
```

Also NCAR's affiliation thing is usually "National Center for Atmospheric Research, Boulder, USA"

**Comment by jbusecke** [comment]
Just pinging @apatlpo again. If you have a few minutes and could send me your approval and affiliation, we can get #232 merged.

**Comment by apatlpo** [comment]
I failed to see all the pings, so sorry about this.

thanks a lot for this kind offer which I won't turn down if you think it is deserved.
please let me know if I can help anyhow with the paper

I apologize in advance for the super long affiliation (crazy french thing):

name: Aurélien  Ponte
ORCID: 0000-0002-0252-6028
Affiliation: Univ. Brest, CNRS, IRD, Ifremer, Laboratoire d’Océanographie Physique et Spatiale (LOPS), IUEM, Brest, France

**Comment by rabernat** [comment]
Yes it is most definitely deserved based on your contributions to xgcm! Glad to have you on board!

**Comment by jbusecke** [comment]
@vrx- I have merged #232. Do you think we can close this issue?

**Comment by vrx-** [comment]
> @vrx- I have merged #232. Do you think we can close this issue?

yes!
